CHANGELOG.md

```diff
@@ -8,10 +8,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* Added validation of the master presence in replicaset and the 
+  master connection to the `utils.get_space` method before 
+  receiving the space from the connection (#331).
 
 ### Changed
 
 ### Fixed
+* Fixed a typo of the fiber status call when
```

Comment:
I don't think that users are interesting in the nature of bug ("typo"), it's more interesting to hear what behavior has been fixed (something about fiber cancel on reload)

Reply:
Done, `typo` -> `bug`, and added  fixed behavior description